XBEE_Radio/XBEE_Radio.ino

```diff
@@ -0,0 +1,48 @@
+int led = 13;
```

Comment:
I'd rather see an example for transmitting rather than receiving, but showing that the Arduino can do both could also be nice.